src/hooks/useLocalStorage.js

```diff
@@ -1,14 +1,20 @@
 import { useState, useEffect } from 'react';
 
-function useLocalStorage(key, initialData) {
-  const [data, setData] = useState(() => {
-    const localData = localStorage.getItem(key);
-    return localData ? JSON.parse(localData) : initialData;
-  });
+function useLocalStorage(storageKey) {
```

Comment:
NON-BLOCKING: This works fine if you don't want to set a default value on initial render. If you want to set an initial default value, you would need to pass in a param for that.